src/main/java/org/sagebionetworks/web/client/widget/team/OpenUserInvitationsWidget.java

```diff
@@ -79,47 +81,67 @@ public void onFailure(Throwable caught) {
 	public void configure(String teamId, Callback teamRefreshCallback) {
 		this.teamId = teamId;
 		this.teamRefreshCallback = teamRefreshCallback;
-		profiles = new ArrayList<UserProfile>();
-		invitations = new ArrayList<MembershipInvtnSubmission>();
-		createdOnDates = new ArrayList<>();
 		currentOffset = 0;
 		getNextBatch();
-	};
+	}
 
 	@Override
 	public void getNextBatch() {
 		synAlert.clear();
```

Comment:
This interaction confuses me.  I would expect that you would need to hide the More button before the rpc.

Reply:
I agree that we should hide the more button before the rpc, but which interaction is confusing other than that?

Reply:
Is it the synAlert.clear() I'm not entirely sure why it was there before the refactor either. I think it's safe to remove it.